docker-cloud/docker-errors-faq.md

```diff
@@ -4,15 +4,9 @@ aliases:
 description: Known Docker Engine issues in Docker Cloud
 keywords:
 - Engine, issues, troubleshoot
-menu:
-  main:
-    parent: docker-cloud
-    weight: 50
-title: Known Issues in Docker Cloud
+title: Known Docker Engine issues in Docker Cloud
```

Comment:
These are not all Engine specific errors (often it's just a collision), so  it doesn't make sense to have "Docker Engine" in the title.